erpnext/manufacturing/doctype/work_order/work_order.js

```diff
@@ -149,6 +149,14 @@ frappe.ui.form.on("Work Order", {
 				frm.add_custom_button(__('Create Job Card'), () => {
 					frm.trigger("make_job_card");
 				}).addClass('btn-primary');
+
+				frm.add_custom_button(__('Start Job Cards'), () => {
+					frm.trigger("start_job_cards");
+				}).addClass('btn-primary');
+
+				frm.add_custom_button(__('Stop Job Cards'), () => {
+					frm.trigger("stop_job_cards");
+				}).addClass('btn-primary');
```

Comment:
Combine these into a category called "Job Card", and use dropdowns to do each action:

```
Job Card:
  - Create
  - Start All
  - Stop All
```

Reply:
Remove the "Job Cards" part from the internal button labels, and keep it as I've shown in the text above